cmd/antrea-controller/controller.go

```diff
@@ -327,6 +331,22 @@ func run(o *Options) error {
 	if features.DefaultFeatureGate.Enabled(features.AntreaPolicy) {
 		go networkPolicyStatusController.Run(stopCh)
 	}
+	if o.config.EnableNodeIPAMController {
```

Comment:
I forget had we discussed to use a featuregate for this. @tnqn

Reply:
I think that we'll always need a disable option for this feature - even after it matures. So do we need a feature gate plus an enable flag?

Reply:
Right, there are other discussions around feature gate or flag. I thought we use feature flag to isolate the code changes we are not confident yet, and also to indicate the phase (alpha, beta, etc.) of a feature. But understood it can overlap with a config flag. Maybe we can remove the feature gate and change to a flag when the feature is graduated to beta?

Like to learn your and others' thoughts.